apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts

```diff
@@ -114,7 +116,18 @@ export function AdministratorFeature({
                 Grant feature
               </button>
             </div>
-            ${featureGrants.length > 0
+            ${featureInConfig != null
+              ? html`
+                  <div class="card-body text-center">
+                    <i
+                      class="fa-solid mr-1 ${featureInConfig
+                        ? 'fa-check text-success'
+                        : 'fa-times text-danger'}"
+                    ></i>
+                    Feature ${featureInConfig ? 'enabled' : 'disabled'} in configuration file
+                  </div>
+                `
```

Comment:
I'd rather render this in the list group, same as any other explicit grants. That makes this case more visually distinct from the empty state.

Reply:
Can you clarify what you mean? I imagine using the same format (`list-group`, non-centered, etc.), but do you mean including the existing entries as well? I'm ok either way, though in that case there should probably be a clear way to indicate a disabled feature which overrides the DB entries.

Reply:
Done in 2097bda96. Grants are listed with text-muted, I hope that's enough.

Reply:
<img width="336" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/1004907/e21d8431-5612-4b7d-a743-a8704077bd72">

Reply:
To make it clear, I only muted in case of config setting. I didn't handle the hierarchy of features, and I'm not planning to do that in this PR.